server/ai_mediaserver.go

```diff
@@ -518,9 +540,14 @@ func kickInputConnection(sourceID string, sourceType string) error {
 		return fmt.Errorf("invalid sourceType: %s", sourceType)
 	}
 
-	resp, err := http.Post(fmt.Sprintf("http://localhost:%s/v3/%s/kick/%s", mediaMTXControlPort, apiPath, sourceID), "", nil)
+	req, err := http.NewRequest(http.MethodPost, fmt.Sprintf("http://%s:%s/v3/%s/kick/%s", mediaMTXHost, mediaMTXControlPort, apiPath, sourceID), nil)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to create kick request: %w", err)
+	}
+	req.SetBasicAuth(mediaMTXControlUser, ls.mediaMTXApiPassword)
```

Comment:
Can we make so that this also works without mediamtx auth configured? The default binding for the control port is localhost anyway.

Reply:
Oh yep that still works, in the mediamtx config I've left in the default localhost user with no password